SparkPost/2web2ui
# AC-1264 - (UI) Single address tab

AC-1264 - (UI) Single address tab

### What Changed
- Tab 1 for the single address form is created
- The form changes endpoints based on Right to Be forgotten or opting out
- Subaccount is assigned to header
- Cypress tests are added for Single Recipient Page

### How To Test
 - Enable 'data_privacy' flag.
 - Navigate to /account/data-privacy/single-recipient.
 - Check if it matched mocks https://sparkpost.invisionapp.com/share/BCVKAJRH4E9#/screens/400849700
 - Submit request for rtbf and Opt out check if requests are sent as doucmented - https://github.com/SparkPost/sparkpost-admin-api-documentation/pull/452/files

### To Do
